@shared/api/helpers/getIconUrlFromIssuer.ts

```diff
@@ -1,5 +1,7 @@
-import StellarSdk, { StellarTomlResolver } from "stellar-sdk";
+import { StellarTomlResolver } from "stellar-sdk";
+
 import { sendMessageToBackground } from "./extensionMessaging";
+import { stellarSdkServer } from "./stellarSdkServer";
```

Comment:
the bulk of this PR is this type of change: swapping out the standard Stellar SDK server for one that automatically configures `allowHttp`, which is needed for users running Horizon locally